packages/appstore/package.json

```diff
@@ -23,17 +17,21 @@
         "deploy:folder": "echo \"No deploy:folder specified\"",
         "deploy:staging": "echo \"No deploy:staging specified\"",
         "deploy:production": "echo \"No deploy:production specified\"",
-        "test:eslint": "eslint \"./src/**/*.?(ts|tsx)\""
+        "test:eslint": "eslint \"./src/**/*.?(ts|tsx)\"",
```

Comment:
@mahdiyeh-fs don't we need to check `js | jsx` as well?

Reply:
Yes @iman-fs. Will add it in.